Package.swift

```diff
@@ -73,6 +77,13 @@ let package = Package(
                 .process("tiles"),
             ]
         ),
+        .target(
+            name: "CarPlayTestHelper",
+            exclude: [
+                "Info.plist",
+                "CarPlayTestHelper.h",
```

Comment:
Good idea to split out this target for the Objective-C-based CarPlay mocking hack.